docs/references/filters.md

```diff
@@ -79,38 +79,19 @@ class FruitFilter:
         return queryset.exclude(name='banana')
 ```
 
-## Overriding the filter method
```

Comment:
Why was this section removed?

Reply:
I thought it was redundant with the previous section title; I don't mind leaving it in though (or we could remove that part from the previous section title)?